From 630d994dbf4f1510b5da05809095a8f95216d222 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Thu, 17 Mar 2022 14:36:04 +0800
Subject: [PATCH 082/274] LF-6254-8 arm64: dts: imx93-11x11-evk: Enable MIPI
 DSI node

This patch enables MIPI DSI node on i.MX93 11x11 EVK platform.

Cc: Sandor Yu <Sandor.yu@nxp.com>
Reviewed-by: Sandor Yu <Sandor.yu@nxp.com>
Signed-off-by: Liu Ying <victor.liu@nxp.com>
---
 arch/arm64/boot/dts/freescale/imx93-11x11-evk.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/imx93-11x11-evk.dts b/arch/arm64/boot/dts/freescale/imx93-11x11-evk.dts
index d35aa6bff..54c427a7b 100644
--- a/arch/arm64/boot/dts/freescale/imx93-11x11-evk.dts
+++ b/arch/arm64/boot/dts/freescale/imx93-11x11-evk.dts
@@ -111,6 +111,10 @@ &dphy {
 	status = "okay";
 };
 
+&dsi {
+	status = "okay";
+};
+
 &lcdif {
 	status = "okay";
 	assigned-clock-rates = <445333333>, <148444444>, <400000000>, <133333333>;
-- 
2.25.1

